algorithms/unit_tests/CMakeLists.txt

```diff
@@ -410,6 +410,13 @@ if(KOKKOS_ENABLE_OPENMPTARGET)
   )
 endif()
 
+# FIXME_OPENMPTARGET - remove sort test as it leads to ICE with clang/17 and above at compile time.
+if(KOKKOS_ENABLE_OPENMPTARGET STREQUAL "Clang" AND KOKKOS_CXX_COMPILER_VERSION VERSION_GREATER 16.0.0)
```

Comment:
Shouldn't it be `VERSION_GREATER_EQUAL 17.0.0 ` in that case?

Reply:
Oh yes you are right. Fixed it